blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx

```diff
@@ -0,0 +1,215 @@
+capabilities, providing per connection SSL and proxy options. Previously such configuration was only possible client wide.
+
+### Un-pooled HTTP connection
+
+Default interactions with an HTTP servers involves providing a request object and let Vert.x HTTP client establish the 
+connection to the actual server and then pool it for subsequent uses.
+
+Sometimes it is desirable to manage such connections instead of reusing pooled connections, for this use case you can use now the Vert.x
+HTTP client [connect](https://vertx.io/docs/5.0.0-SNAPSHOT/apidocs/io/vertx/core/http/HttpClientAgent.html#connect(io.vertx.core.http.HttpConnectOptions))
+capabilities.
+
+### Micrometer Metrics performance improvements
+
+Our integration of Micrometer Metrics overhead has been reduced, you can learn more about that in this blog [post](https://vertx.io/blog/micrometer-metrics-performance/).
+
+### TCP shutdown
+
+HTTP server and client support graceful shutdown.
+
+You can shut down a server or a client.
+
+Calling `shutdown` initiates the shut-down phase whereby the server or client are given the opportunity to perform clean-up actions.
+
+- A standalone HTTP server unbinds
+- A shared HTTP server is removed from the set of accepting servers
+- An HTTP client refuses to send any new requests
+
+When all connections inflight requests are processed, the server or client is then closed.
+
+### Application Launcher
+
+Application [launcher](https://vertx.io/docs/5.0.0-SNAPSHOT/vertx-launcher-application/java/) is a new tool which addresses the
+concern of launching a Vert.x application packaged as a far jar. It replaces the Vert.x 4 `io.vertx.core.Launcher`.
+
+## Sunsets and removals
+
+A few components are sunsetting in 5.0, that means they are still supported for the lifetime of the 5.x series,
+but we don't encourage using them anymore since we provide replacements for them. Such components are scheduled
+to go away in the next major release (Vert.x 6).
+
+Here is the list of components we sunset in 5.x
+
+| Component         | Replacement                            |
+|-------------------|----------------------------------------|
+| gRPC Netty        | Vert.x gRPC client and server          |
+| JDBC API          | SQL client API implementation for JDBC |
+| Service Discovery | Vert.x Service Resolver                |
+| RxJava 1          | Mutiny or RxJava 3                     |
+| OpenTracing       | OpenTelemetry                          |
+| Vert.x Unit       | Vert.x JUnit 5                         |
+
+Here is the list of components removed in 5.x, that were sunset in 4.x series.
+
+- Vert.x Sync, replaced by Vert.x virtual threads 
+- Service Factories / Maven Service Factory / HTTP Service Factory 
+
+## Finally
+
+The [deprecations and breaking changes](https://github.com/vert-x3/wiki/wiki/5.0.0-Deprecations-and-breaking-changes)
+ can be found on the wiki.
+
+For this release there are no Docker images.,
```

Comment:
```suggestion
For this release there are no Docker images.
```

Perhaps we should move this up to the sunset-ted components list.

Reply:
yes

Reply:
will do, it's left over from the past